server/etcdserver/api/v3alarm/alarms.go

```diff
@@ -59,16 +59,10 @@ func (a *AlarmStore) Activate(id types.ID, at pb.AlarmType) *pb.AlarmMember {
 		return m
 	}
 
-	v, err := newAlarm.Marshal()
-	if err != nil {
-		a.lg.Panic("failed to marshal alarm member", zap.Error(err))
-	}
-
-	b := a.bg.Backend()
-	b.BatchTx().Lock()
-	b.BatchTx().UnsafePut(buckets.Alarm, v, nil)
-	b.BatchTx().Unlock()
-
+	tx := a.bg.Backend().BatchTx()
+	tx.Lock()
```

Comment:
Seems it should be  (safe) `buckets.MustPutAlarm(a.lg, tx, newAlarm)`

Reply:
Done